src/utils/keyPairGenerator.js

```diff
@@ -68,10 +68,10 @@ const threadJobWorkerSeed = (
 ) => {
   return () => {
     return new Promise((resolve, reject) => {
-      let count = 20;
-      let lastCount = connectionsCount + (threadIndex * 20);
+      let count = 10;
+      let lastCount = lastConnectionKeyIndex + (threadIndex * count);
       if (lastConnectionKeyIndex < 0) {
-        count = Math.ceil((100 + connectionsCount) / 5);
+        count = Math.ceil((50 + connectionsCount) / 5);
```

Comment:
It would be good to extract those magic numbers (10, 20, 50, 5) into some constants